lib/Test/Mojo.pm

```diff
@@ -337,6 +361,13 @@ sub websocket_ok {
   return $self->_request_ok($self->ua->build_websocket_tx(@_), $_[0]);
 }
 
+sub _attr {
+  my ($self, $selector, $attribute) = @_;
+  return '' unless my $e = $self->tx->res->dom->at($selector);
+  return '' unless my $a = $e->attr($attribute);
+  return $a;
```

Comment:
`$a` is a reserved variable in Perl.

Reply:
Yeah, I know ;-)
Added commit to fix this.